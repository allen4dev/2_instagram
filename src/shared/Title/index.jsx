import React from 'react';
import { string } from 'prop-types';

import './index.css';

function Title({ children }) {
  return <h2 className="Title">{children}</h2>
}

Title.propTypes = {
  children: string.isRequired,
};

export default Title;
