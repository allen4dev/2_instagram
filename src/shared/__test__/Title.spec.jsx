import React from 'react';
import { shallow } from 'enzyme';

import Title from './../Title';

test('should render logo', () => {
  const logo = shallow(<Title>Dummie Test</Title>);
  expect(logo).toMatchSnapshot();
});
