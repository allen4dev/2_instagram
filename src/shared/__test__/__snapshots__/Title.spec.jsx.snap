// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render logo 1`] = `
<h2
  className="Title"
>
  Dummie Test
</h2>
`;
